content/projects/gsoc/2020/project-ideas/jenkins-distribution-customize-service.adoc

```diff
@@ -43,6 +43,23 @@ A readable API documentation is
 necessary. Like many projects, the Swagger UI is a good example.
 We can start the backend project with SpringBoot, it might save a lot of time.
 
+== Quickstart
+
+The link:https://github.com/jenkinsci/custom-war-packager[Custom WAR Packager for Jenkins] is a good starting point.
+
+* Clone the repository and build the project, Try using the `all-latest-core` folder inside the demo folder and generate your own war.
```

Comment:
This is a bad demo to start from. Better to use other demos or projects like https://github.com/jenkinsci/ci.jenkins.io-runner or https://github.com/librecores/librecores-ci-jenkins-server which are based on CWP

Reply:
Okay, I put in the quick-start because it was mentioned in the document and thought it would be a good start to learn the basics. Thanks for the suggestions. Will make the changes.